hashicorp_vault.json

```diff
@@ -5,23 +5,25 @@
     "publisher": "Splunk Community",
     "type": "generic",
     "main_module": "hashicorp_vault_connector.py",
-    "app_version": "1.0.1",
+    "app_version": "1.1.0",
     "utctime_updated": "2021-11-10T00:20:20.000000Z",
     "package_name": "phantom_hashicorp_vault",
     "product_vendor": "Dallan",
     "product_name": "Hashicorp Vault",
     "product_version_regex": ".*",
     "python_version": "3",
-    "min_phantom_version": "4.9.39220",
+    "fips_compliant": true,
```

Comment:
since this appears to be a community app, I'm not sure if this was added by the contributor or us. did we verify this apps is actually FIPS compliant? if we're not certain / can't confirm this flag is not required for community apps

Reply:
As per mentioned here - https://splunk.slack.com/archives/GBSGHUUD9/p1639635017100200, this app does not use hashlib library. Hence, marked it as true. Also, without adding the flag, the pipeline does not pass.

Reply:
> Also, without adding the flag, the pipeline does not pass.
the FIPS flag check should only run for "splunk-supported" apps. since this app is "developer-supported" it doesn't run

if you search the static test report you'll see it doesn't mention FIPS

Reply:
Removed.